packages/simple-cache-provider/index.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * Copyright (c) 2013-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ */
+
+'use strict';
+
+const SimpleCacheProvider = require('./src/SimpleCacheProvider');
+
+// TODO: decide on the top-level export form.
+// This is hacky but makes it work with both Rollup and Jest.
+module.exports = SimpleCacheProvider.default
+  ? SimpleCacheProvider.default
```

Comment:
Let’s not duplicate this hack anymore. It’s only necessary for legacy stuff.

See Brian’s react-is package for how we should do it in the future.